worldedit-fabric/src/main/java/com/sk89q/worldedit/fabric/FabricWorldEdit.java

```diff
@@ -286,6 +287,7 @@ private void onStopServer(MinecraftServer minecraftServer) {
         WorldEdit worldEdit = WorldEdit.getInstance();
         worldEdit.getSessionManager().unload();
         WorldEdit.getInstance().getEventBus().post(new PlatformUnreadyEvent(platform));
+        WorldEdit.getInstance().getExecutorService().shutdown();
```

Comment:
If this is something that must be shut down on Fabric, I feel that we should probably also shut this down on other platforms.